src/libs/compound-v3/utils.ts

```diff
@@ -0,0 +1,62 @@
+import { BigNumber } from 'ethers';
+import BigNumberJS from 'bignumber.js';
+import { SECONDS_PER_YEAR } from './constants';
+import * as common from '@protocolink/common';
+import * as logics from '@protocolink/logics';
+
+export function getMarketLabel(chainId: number, marketId: string) {
+  if (chainId === common.ChainId.arbitrum && marketId === logics.compoundv3.MarketId.USDC) {
+    return 'USDC.e';
+  }
+  return marketId;
+}
+
+export function calcApr(rate: BigNumber) {
+  return common.toBigUnit(rate.mul(SECONDS_PER_YEAR), 18, { displayDecimals: 4 });
+}
+
+export function calcUtilization(borrowCapacityValue: string | BigNumberJS, borrowValue: string | BigNumberJS) {
+  let utilization = '0';
+  borrowCapacityValue = new BigNumberJS(borrowCapacityValue);
```

Comment:
Do you need to check if the input value is negative as well?

Reply:
As discussed earlier, this function is purely for calculation purposes. The validation part, however, will be handled externally.